node/clients/checkout.ts

```diff
@@ -21,12 +21,12 @@ export class Checkout extends JanusClient {
   }
 
   private getCommonHeaders = () => {
-    const { orderFormId } = this.context as CustomIOContext
+    const { orderFormId, segmentToken, sessionToken } = this.context as CustomIOContext
     const checkoutCookie = orderFormId ? checkoutCookieFormat(orderFormId) : ''
+    const segmentTokenCookie = segmentToken?  `vtex_segment=${segmentToken};` : ''
```

Comment:
maybe adding a space in here ?
```suggestion
    const segmentTokenCookie = segmentToken ?  `vtex_segment=${segmentToken};` : ''
```